static/src/js/components/OrderStatus/OrderStatus.jsx

```diff
@@ -184,7 +185,7 @@ export const OrderStatus = ({
             <Well.Heading>Download Status</Well.Heading>
             <Well.Introduction>{introduction}</Well.Introduction>
             {
-              (isLoading && !isLoaded) && (
+              (!id && !isLoaded) && (
```

Comment:
I think a better way to ensure this skeleton shows up would be defaulting `isLoading` to `true` on line 66.

What you have works, but we have a variable specifically for this in `isLoading`, using `!id` isn't as clear. 

There is also another Skeleton that should be showing up on line 223.

Reply:
I think it was undefined because `retrieval` is defaulted to `{}` until the request finishes.

Reply:
After a huddle we found a better way to do this, so I'll resolve this comment